lib/matplotlib/dviread.py

```diff
@@ -704,29 +733,27 @@ def _mul2012(num1, num2):
 
 class Tfm(object):
     """
-    A TeX Font Metric file. This implementation covers only the bare
-    minimum needed by the Dvi class.
-
-    .. attribute:: checksum
+    A TeX Font Metric file.
 
-       Used for verifying against the dvi file.
-
-    .. attribute:: design_size
-
-       Design size of the font (in what units?)
+    This implementation covers only the bare minimum needed by the Dvi class.
 
-    .. attribute::  width
+    Parameters
+    ----------
+    filename : string or bytestring
 
+    Attributes
+    ----------
+    checksum : int
+       Used for verifying against the dvi file.
+    design_size : int
+       Design size of the font (unknown units)
+    width : dict
```

Comment:
Assuming they are all of similar description and the original just eschewed repeating all that, this can be combined with the next two as:
```
width, height, depth : dict
    Dimensions of each character, ...
```